applications/search-api/src/main/java/no/dcat/portal/query/DatasetsQueryService.java

```diff
@@ -149,8 +152,12 @@
 
             @ApiParam("Filters on catalog. ")
             @RequestParam(value = "catalog", defaultValue = "", required = false)
-                    String catalog) {
+                    String catalog,
 
+            @ApiParam("Filters on returnFields. ")
```

Comment:
En bedre beskrivelse er noe slikt som "Commaseparated list of which fields should be returned. E.g id,title,description"

Reply:
fikset